index.html

```diff
@@ -400,6 +400,21 @@ <h3>Timeline</h3>
 
 			<section id="success-criteria">
 				<h2>Success Criteria</h2>
+				
+				<p>The EPUB 3 Working Group’s work is considered a success 
+				if all of the following conditions are met:</p>
+				
+				<ol>
+					<li>There are multiple, independent, interoperable implementations 
+					of EPUB 3.X that are widely used.</li>
+					<li>Existing content is supported. All conforming EPUB 3.2 publications should remain conformant to EPUB 3.X.</li>
+					
+					<li>Changes to the global ecosystem 
+					(including authors, publishers, distributors, retailers, reading systems, and tool makers) 
+					are minimized, and made only after careful consideration 
+					of how they would affect all constituencies.</li>
+					<li>Validation tools such as EPUBCheck fully support EPUB 3.X.</li>
+				</ol>
```

Comment:
I understand and agree with the goal, but I am worried about putting this into the charter. Whether epubcheck will evolve or not, whether there will be other tools out there or not, etc., are not under the control of this Working Group; we all know it is a matter of funding, of having the right people doing that work, etc. Putting such a requirement into the charter may not be appropriate.

Reply:
Heads up: we may get remarks from the AC during vote that this is a repetitive set of terms, which are already expressed in the scope section (and we often get comments on making the charter as succinct as possible). I wonder whether it is not a better place to put these items into the scope section.

Reply:
It's complicated by timelines, too. Updating of epubcheck typically occurs after the spec is finalized so that there's no wasted effort trying to implement features that then go away.

But, the flip side is revisions go nowhere until epubcheck is updated as distribution is more tightly controlled than the web. So I wouldn't discount the lack of traction a revision will have if the only indicator of success is two implementations.

Reply:
> Whether epubcheck will evolve or not, whether there will be other tools out there or not, etc., are not under the control of this Working Group; we all know it is a matter of funding, of having the right people doing that work, etc.

Do you think EPUB 3.X could be successful if it is not supported by EPUBCheck? I think that question was answered by EPUB 3.1. The community is concerned about the future of EPUB, and we should address that as best we can. Many things in the charter are not controlled by the working group, including implementations :)

Reply:
I agree that EPUBcheck is hugely important, but to make it some sort of a bottleneck in the charter is what I am not sure about. Say we have 55 implementations, all pass the bars, but epubcheck gets delayed by a few months because, say, the main implementer has some personal issues. Do we suspend the group and not publish the Rec?

Reply:
> Do we suspend the group and not publish the Rec?

The charter isn't only about going to REC, is it? Isn't it also a general statement of all the things the group expects to achieve to be successful?

There's also no reason why this group couldn't help submit patches for epubcheck as part of the process. Why limit ourselves to creating RS tests when we change things? It would make the chore of patching epubcheck so much simpler if issues/PRs were submitted as we went instead of having to comb the commit logs afterwards. That's partly why we end up not realizing things that should have been allowed/disallowed have gone unchecked for years and then become problematic to correct.

Reply:
Ok. I have my bad feeling about this, but I won't object if it makes everybody happy...

Reply:
What if we move it out of the bullet list and into the last paragraph:

"To promote interoperability, all changes made to specifications should have tests. Changes that affect the validity of EPUB publications should also be coordinated with the epubcheck tool (e.g., new issues and/or pull requests)."

Would that make it less of an awkward fit? It would clearly identify the nature of the coordination we infer by listing the epubcheck community in section 4.

Reply:
> Say we have 55 implementations, all pass the bars, but epubcheck gets delayed by a few months because, say, the main implementer has some personal issues. Do we suspend the group and not publish the Rec?

I know it sounds extreme, but I think we leave it in CR until all the pieces are in place. I don't think the existence of a REC is the primary goal, or at least we shouldn't sacrifice other goals just to say we have a REC. I think that's what some people are fearing--that we focus so much on getting through the process that we forget why we are going through the process. The goal is a better ecosystem; an EPUB REC is just one component of that.